### DISCUSSION TITLE
"apt upgrade" command gives an error "404  Not Found [IP: 14.234.210.38 443]"

### DISCUSSION THREAD

**Discussion by Invincible25** [termux/termux-packages#10363] [General]
I usually do "apt upgrade" command to Termux before installing a new package and it works as usual.
But today when I gave the command, it gave me an error, See the below ScreenShot:

![Screenshot_2022-04-27-20-21-17-786](https://user-images.githubusercontent.com/104266241/165546944-53587f50-45e7-4f5c-8a1c-46b9cfcf3d3e.jpeg)

It was working fine until I connected a VPN and after disconnecting the VPN, this problem came to me. 

I am not sure but MAYBE the VPN caused this problem🤔 Anyway, It's just a guess.

PLEASE HELP ME AND ALSO THANKS FOR YOUR TIME

**Comment by agnostic-apollo** [comment]
Supposed to use `apt update`

**Reply by chrisspen** [reply]
`apt update` gives me the exact same 404 not found errors. Is Termux no longer maintained? Why are all the repos missing?

**Reply by TomJo2000** [reply]
The Termux repositories are actively maintained, as [this git repo's commit history](https://github.com/termux/termux-packages/commits/master) can attest to.

It's likely that this was a temporary issue with one of the repository mirrors.
Which, seeing as this issue was posted 9 months ago, and I've since used the kubeterm mirror, has been resolved.

**Reply by chrisspen** [reply]
Yeah, I managed to fix it by using `termux-change-repo` to switch to a different mirror. I've since updated all the packages, and now it's not showing me what the original default mirror was (I think it was Bintray), but I switched to the Grimler mirror, which worked fine.

**Reply by sylirre** [reply]
Bintray repo is set by default in Termux from Play Store. Yes, the Play Store edition is no longer maintained because it just can't be maintained (sounds ridiculous but that's true).

F-Droid should be used instead: https://f-droid.org/packages/com.termux/

**Comment by chrisspen** [comment]
What makes the Play Store unmaintainable?

At the very least, there should be a note in the README to change the mirror. I stumbled across the F-Droid fix myself just searching some blogs, but it took some time.

**Reply by truboxl** [reply]
If you read [README.md](https://github.com/termux/termux-packages#readme), we have mentioned:
>Quick how-to about Termux package management is available at [Package Management](https://github.com/termux/termux-packages/wiki/Package-Management).
**It also has info on how to fix repository is under maintenance or down errors when running apt or pkg commands.**

And clicking the link will take you to the Wiki page explaining why Play Store is unmaintainable

We also pin issues related to Play Store deprecation in the Issues page: https://github.com/termux/termux-packages/issues

Maybe you need to give us the search queries for the search engine and let us see how we influence SEO

**Reply by sylirre** [reply]
There 2 reasons why it is unmaintainable:
1. We don't have access to Play Store account. Sounds ridiculous, but Termux founder abandoned the project and now Termux has different maintainers. We never granted even the limited access to the account, requests to unpublish Termux app from Play Store were ignored.
2. Target SDK level required by Play Store is unacceptable for Termux. https://github.com/termux/termux-packages/wiki/Termux-and-Android-10

> I stumbled across the F-Droid fix myself just searching some blogs

I always wondered why people search various third party blogs first instead of official resources and community pages. The information is available at least at GitHub, Termux Wiki and Reddit.
